src/components/apps/toolbar/AppNavigation.js

```diff
@@ -124,12 +130,32 @@ function AppNavigation(props) {
             categoryList = categoryList.concat(allAppsCategory());
             setCategories(categoryList);
             handleAppNavError(null);
+            // If a user has just logged in, they may still have one of the fake
+            // categories given to the anonymous user selected with a fake
+            // UUID
+            if (
+                [constants.FAV_APPS, constants.APPS_UNDER_DEV].includes(
+                    selectedCategory?.id
+                )
+            ) {
+                handleCategoryChange(
+                    categoryList.find(
+                        (category) => category.name === selectedCategory.name
```

Comment:
Here's where the `AppNavigation` component updates the currently selected category with the real category ID instead of the fake category ID.